docs/development/build-instructions-gn.md

```diff
@@ -72,6 +72,32 @@ $ gclient sync --with_branch_heads --with_tags
 # This will take a while, go get a coffee.
 ```
 
+#### A note on pulling/pushing
+
+If you intend to `git pull` or `git push` from the official `electron`
+repository in the future, you now need to update the respective folder's
+origin URLs.
+
+```sh
+$ cd src/electron
+$ git remote remove origin
+$ git remote add origin https://github.com/electron/electron
+$ cd -
+```
+
+:memo: `glient` works by checking a file called `DEPS` inside the 
+`src/electron` folder for dependencies (like Chromium or Node.js).
+Running `gclient sync -f` ensures that all dependencies required
+to build Electron match that file.
+
+So, in order to pull, you'd run the following commands:
+```sh
+$ cd src/electron
+$ git pull
+$ cd ..
```

Comment:
There is no need to switch folders here.
`gclient` looks for a config in all ancestral folders.